internal/injector/builtin/yaml/civisibility/testing.yml

```diff
@@ -0,0 +1,212 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2023-present Datadog, Inc.
+---
+# yaml-language-server: $schema=../../../../../docs/static/schema.json
+meta:
+  name: go-testing
```

Comment:
Currently in all other tracers we have CI Visibility stuff kind of isolated, for a couple of reasons: First it changes the mode of the tracer behaviour, other writers, other type of payloads, different transports, etc, so is not a simple new integration for the tracer it self. The second because we modify a lot of the behaviour we tend to group all this to a single folder and be friendly with the codeowners file. I did the same because this will be the first integration under the CI Visibility umbrella but I don't think it will the only one.

Reply:
That makes sense to me... But I'd still just name this `testing` (Orchestrion only touches go, so it's a bit of a stutter)

```suggestion
  name: testing
```

Reply:
done in [92ee768](https://github.com/DataDog/orchestrion/pull/340/commits/92ee768360c07a61c2311485193b2236c93be0b1)